frontend/src/schemas/SchemaSelect.tsx

```diff
@@ -12,21 +12,23 @@ import Loading from 'src/common/Loading'
 import Link from 'src/Link'
 import MessageAlert from 'src/MessageAlert'
 import SchemaButton from 'src/schemas/SchemaButton'
-import { EntryInterface, EntryKind, EntryKindLabel, SchemaInterface, SchemaKind } from 'types/types'
+import { EntryInterface, EntryKind, SchemaInterface, SchemaKind, SchemaKindKeys, SchemaKindLabel } from 'types/types'
 
 type SchemaSelectProps = {
-  entry: EntryInterface
+  schemaKind: SchemaKindKeys
+  entry?: EntryInterface
```

Comment:
I would prefer to keep `entry` a required prop as we need this to determine where to route the user to if the click the back button. It's also needed to know what text to put on that button. This would also mean that we don't need `id` as a prop.

I suggest updating `frontend/pages/model/[modelId]/access-request/schema.tsx` to fetch the model so that it can be passed into this component.